tests/test_units/test_rms_context.py

```diff
@@ -554,6 +554,32 @@ def test_gridproperty_export_file_set_name(rmssetup, rmsglobalconfig, gridproper
     )
 
 
+@inside_rms
+def test_gridproperty_export_with_geometry(
+    rmssetup, rmsglobalconfig, grid, gridproperty
+):
+    """Export the the grid and the gridprop(s) with connected geometry."""
+    logger.info("Active folder is %s", rmssetup)
+    os.chdir(rmssetup)
+
+    edata = dataio.ExportData(
+        config=rmsglobalconfig, content="depth"
+    )  # read from global config
+    grid_output = edata.export(grid, name="MyGrid")
+
+    # Here the export of the grid property point to the path of the exported geometry,
+    # which is then sufficient to provide a geometry tag in the data settings.
+    output = edata.export(gridproperty, name="MyGridProperty", geometry=grid_output)
```

Comment:
probably should design the test here using a new instance of `ExportData` with content `cpgrid_property` to export the gridproperty, this pattern will hopefully start to give deprecation warnings soon.

Reply:
Edit ignore the content part in my comment above 😄 But probably we want to keep the export in two separate ExportData instances.